Source/Localization/es-UY.lproj/Generated.strings

```diff
@@ -29,8 +29,8 @@
 "Text.usersInYourNetwork" = "Users in your network";
 "Text.goToYourNetwork" = "Go to Your Network";
 "Text.emptyHomeFeedMessage" = "And it is rather bare! Have you considered following a few users or topics?";
-"Text.startedFollowing" = "{{somebody}} empezó a seguir a";
-"Text.stoppedFollowing" = "{{somebody}} dejo de seguir a";
```

Comment:
@martindsq I'm not sure why this changed, but I also don't see the Spanish translations for these showing up in Crowdin. Can you confirm they are correct and maybe set the translations [there](https://crowdin.com/translate/planetary/330/en-esuy?filter=basic&value=3#q=started%20following).